lib/ansible/config/module_defaults.yml

```diff
@@ -664,15 +664,15 @@ groupings:
   - os
   os_coe_cluster_template:
   - os
-  os_flavor_facts:
+  os_flavor_info:
```

Comment:
Please retain the previous name since module_defaults doesn't use aliases otherwise. I'm adding the rest back in #61203.

Reply:
Thanks! Fixed in 04af3b47c86904b2282e2430a67adadbb78c2eed